spec/models/user_spec.rb

```diff
@@ -416,20 +416,25 @@
   end
 
   describe "self.search" do
-    it "find users by email" do
-      user1 = create(:user, email: "larry@consul.dev")
-      create(:user, email: "bird@consul.dev")
-      search = User.search("larry@consul.dev")
+    describe "find users" do
+      let(:larry) { create(:user, email: "larry@consul.dev", username: "Larry Bird") }
+      let(:robert) { create(:user, email: "robert@consul.dev", username: "Robert Parish") }
```

Comment:
Since `robert` is never used:

```suggestion
      before { create(:user, email: "robert@consul.dev", username: "Robert Parish") }
```